reactor/runner.js

```diff
@@ -0,0 +1,61 @@
+// =============== FILL THE DATA IN THIS AREA ===============
+var apiKey = 'INSERT YOUR OPERATOR API KEY HERE';
+
+// UNCOMMENT ONE PARTICULAR EVENT FUNCTION YOU ARE USING
+// AND FILL IT USING THE TEMPLATES
```

Comment:
same as documentation: I would put some data on the action/product/thng object, or have a link here in the comment to the reactor documentation, specially the runner.js part.